src/modules/number/index.ts

```diff
@@ -125,30 +120,29 @@ export class NumberModule extends SimpleModuleBase {
       | number
       | {
           /**
-           * Lower bound for generated number.
+           * Lower bound for generated number, inclusive. Defaults to `0.0`.
```

Comment:
The default is already below.

```suggestion
           * Lower bound for generated number, inclusive.
```